src/status_im2/subs/chat/messages.cljs

```diff
@@ -267,3 +268,13 @@
                           joined
                           loading-messages?)
            (albumize-messages))))))
+
+(re-frame/reg-sub
+ :messages/resolve-mention
+ (fn [[_ identity] _]
```

Comment:
I would just recommend not using the word `identity` because it's a core clj function. It confused me a little bit when reading for the first time.